pkg/runtime/from_config.go

```diff
@@ -59,34 +66,37 @@ func (r *Runtime) setEvaluatorFromConfig(logger *logger.Logger) error {
 }
 
 func (r *Runtime) setSyncImplFromConfig(logger *logger.Logger) error {
+	regCrd := regexp.MustCompile(crdRegex)
```

Comment:
General practice for these regexp compiles is to do them in an `init()` func but I'll leave that decision to you

Reply:
I've moved these to an `init()` function, thanks for the suggestion